src/coreclr/nativeaot/docs/android-bionic.md

```diff
@@ -0,0 +1,30 @@
+# Using Native AOT with Android Bionic
+
+Starting with .NET 8 Preview 7, it's possible to build shared libraries and command line executables for Android.
+
+Not a full Android experience is available - it's only possible to publish for two Bionic RID: bionic-arm64 and bionic-x64. Publishing for Android RIDs (android-arm64/android-x64) is not possible. This limited experience corresponds to building with [Android NDK](https://developer.android.com/ndk) from Native code - the limitations are similar. Interop with Java needs to be done manually through JNI, if necessary.
+
+To build for Bionic:
+
+* Ensure you have [Android NDK](https://developer.android.com/ndk/downloads) for your system downloaded and extracted somewhere.
+* Update your PATH to include `{NDK_ROOT}\toolchains\llvm\prebuilt\{OS_ARCH}\bin`. Replace `{NDK_ROOT}` with the path where you extracted the NDK, and replace `{OS_ARCH}` with your host OS and architecture (e.g. windows-x86_64 or linux-x86_64). Make sure this entry is first in your path - we need to make sure that running `clang` will execute `clang` from this directory.
+* You can either create a new project or use an existing project. For this guide, let's create a new one:
+  ```sh
+  $ dotnet new console -o HelloBionic --aot
+  $ cd HelloBionic
+  ```
+  Note: the `--aot` parameter added `<PublishAot>true</PublishAot>` to the new project.
+* Publish the project:
+  ```sh
+  $ dotnet publish -r linux-bionic-arm64 -p:DisableUnsupportedError=true -p:PublishAotUsingRuntimePack=true
```

Comment:
Are the two properties going away eventually? If not, we should probably at least mention them in this doc. Their names don't make it super clear what they're doing.

Reply:
The `PublishAotUsingRuntimePack` is supposed to go away. We currently have to set this for iDevices for the same reason. We'd like to use the runtime pack for all configurations, not just iDevices/Bionic. There's an 8.0 issue on it.